tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -2542,33 +2542,32 @@ bool CudnnSupport::DoConvolveImpl(
   //   GetCudnnConvolutionForwardAlgorithm().
   if (algorithm_config.algorithm().is_default()) {
     // With the default algorithm, use Cudnn's heuristics.
-    auto get_algorithm =
-        [&](bool specify_limit) SHARED_LOCKS_REQUIRED(dnn_handle_mutex_) {
-          cudnnConvolutionFwdPreference_t preference =
-              specify_limit ? CUDNN_CONVOLUTION_FWD_SPECIFY_WORKSPACE_LIMIT
-                            : CUDNN_CONVOLUTION_FWD_NO_WORKSPACE;
-
-          auto memory_limit_bytes =
-              scratch_allocator == nullptr
-                  ? 0
-                  : scratch_allocator->GetMemoryLimitInBytes(stream);
-          if (memory_limit_bytes < 0) {
-            memory_limit_bytes = 0;
-          }
+    auto get_algorithm = [&](bool specify_limit) SHARED_LOCKS_REQUIRED(
```

Comment:
Can you remove the whitespace changes from this PR? It makes it difficult to see what was really changed.

Note I find [Meld](http://meldmerge.org/) useful for reverting a subset of changes.

Reply:
For future reference, I would recommending using git-clang-format, which formats only modified lines. I'm not sure if it can integrate with your editor, though.

Reply:
Thanks, I reverted the changes. I wasn't aware of git-clang-format and will check to see if I can integrate it to my editor.